### PR TITLE
[Binding addons starting with "R" v2] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: addons/binding/org.openhab.binding.rme/README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 All devices support some of the following channels:
 
-| Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
-| waterlevel | Number  | Indicates the % the cistern is filled with water |
-| mode | String | Indicates the operation mode of the RME Rain Manager, Automatic or Manual |
-| source | String | Indicates water source used to supply water, e.g cistern or city |
-| exitpump | Switch | Indicates whether the exit pump is switched on |
-| entrypump | Switch | Indicates whether the entry pump is switched on |
+| Channel Type ID | Item Type | Description                                                               |   |   |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.rme/README.md]: Remove columns's

File: addons/binding/org.openhab.binding.russound/README.md [hunk 2]
```diff
@@ -36,144 +49,146 @@ The following configurations occur for each of the bridges/things:
...
-| mmcontextmenu      | W          | Switch****   | Whether to initial a media management context session (ON) or close an existing one (OFF) |
+
+| Channel Type ID    | Read/Write | Item Type  | Description                                                                               |
+|--------------------|------------|------------|-------------------------------------------------------------------------------------------|
+| name               | R          | String     | The name of the zone (changed by SCS-C5 software)                                         |
+| source             | RW         | Number     | The (physical) number for the current source                                              |
+| bass               | RW         | Number     | The bass setting (-10 to 10)                                                              |
+| treble             | RW         | Number     | The treble setting (-10 to 10)                                                            |
+| balance            | RW         | Number     | The balance setting (-10 &lsqb;full left&rsqb; to 10 &lsqb;full right&rsqb;)                                  |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.russound/README.md]: I see an exception in line width here, in which order did you do the fixes?

Comment (bgilmer77) [reply addons/binding/org.openhab.binding.russound/README.md]: @martinvw maybe you can help me understand something.  I was getting a linter error caused by left and right square brackets.  It seems that the linter thinks these are references to a note somewhere else in the article as in:

```java
Wikipedia[1] is a great site!

[1] https://www.wikipedia.org
```
So I escaped the left and right brackets with `&lsqb;` and `&rsqb;`  So the one line of the table is now longer than all the others.  I assumed that when it is rendered by Jekyll, the substitutions would be made and the line would then be the correct length.  But perhaps I am wrong? 
WDYT?

Comment (martinvw) [reply addons/binding/org.openhab.binding.russound/README.md]: I assume that we do the formatting to make the text variant look great, so just treat it as 5 chars.

File: addons/binding/org.openhab.binding.rme/README.md (hunk 1 above)
Comment (bgilmer77) [reply addons/binding/org.openhab.binding.rme/README.md]: Agree - will address in next commit.

File: addons/binding/org.openhab.binding.russound/README.md (hunk 2 above)
Comment (bgilmer77) [reply addons/binding/org.openhab.binding.russound/README.md]: @martinvw I am not clear.  Should I take the `&lsqb` out and put a "`[`" in?

Comment (martinvw) [reply addons/binding/org.openhab.binding.russound/README.md]: No `&lsqb` is fine but just treat it as the width it has, I assume that is the default behaviour if your reformat it after having replaces them by `&lsqb`

**Pull request merged**